content/docs/command-reference/dag.md

```diff
@@ -53,20 +53,20 @@ is not available (e.g. on Windows), the output is simply printed out.
 ### Providing a custom pager
 
 It's possible to override the default pager via the `DVC_PAGER` environment
-variable. For example, the following command will replace the default pager with
-[`more`](<https://en.wikipedia.org/wiki/More_(command)>), for a single run:
+variable. Set it to a program found in `PATH` or give a full path to it. For
+example on Linux shell:
 
 ```dvc
```

Comment:
Q is do we want to add a task in https://github.com/iterative/dvc.org/issues/2939 for Windows tabs? I don't think it's that relevant for this. Could also just link to some env var config doc from MS Windows.

Reply:
e.g. https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1
or something from https://support.microsoft.com/search/results?query=environment+variables+WINDOWS&isEnrichedQuery=true -- it's ridiculously hard to find official Windows docs...

Reply:
It's should be like `set DVC_PAGER=more` but I didn't test on Windows CMD. Do we have  some ballpark estimate of the Windows CMD users? I don't expect them to be more than 5% to be honest.

Reply:
I also don't think this one is worth a Windows tab. I'm just thinking about linking to Windows env var official docs but I can't find it 🤷🏼  so we can just not mention Windows here.

Reply:
40% of `dvc.org/doc` viewers are on Windows.

Reply:
But very few of those 40% use Windows `cmd.exe` or PowerShell to use `dvc`.